### PR TITLE
Issue#9

### LINKED ISSUE #9: Backend endpoint that fetches data from five remote servers
Hints:
https://docs.google.com/document/d/1C6tAOaZ75OTHKTf17uUF0PAysJ409KXBNnTME7f966M/edit#heading=h.v7d40c3v1ucc

### PR DESCRIPTION
**Pull request by Castau**
Calling 5 apis and returns a map with the specific api as key and what it returns as value. The api used is the SWAPI suggested in the assignment (https://swapi.co/api/), and the specific endpoints (and keys in the returned map) are:

* people
* planets
* starships
* vehicles
* species

Please tell me if you want it done differently, I wasn't sure what the best solution would be. The data isn't coming from five different serves, they are all from the same (suggested in the assignment) API. And for now they are not specified as parameters in the method, but just hardcoded, I don't know if this it what they meant. 
The testMain can be deleted when the testing is complete.
Fixes #9

### CODE REVIEW

File: src/main/java/facades/ApiFacade.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import java.util.Queue;
+import java.util.Scanner;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeoutException;
+import javafx.util.Pair;
```
Comment (Runi-VN) [inline src/main/java/facades/ApiFacade.java:21]: Never seen this `javafx.util.Pair` before. Nice.

File: src/main/java/rest/SWAPIRessource.java [hunk 2]
```diff
@@ -35,36 +38,11 @@ public String getInfoForAll() {
...
     }
 
-    @GET
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("admin")
-    @RolesAllowed("admin")
-    public String getFromAdmin() {
-        String thisuser = securityContext.getUserPrincipal().getName();
-        return "{\"msg\": \"Hello to (admin) User: " + thisuser + "\"}";
-    }
```
Comment (Runi-VN) [inline src/main/java/rest/SWAPIRessource.java]: I dont necessarily agree with the deletions here, they are needed for #8

File: src/main/java/rest/SWAPIRessource.java [hunk 3]
```diff
@@ -35,36 +38,11 @@ public String getInfoForAll() {
...
-        }
-    }
-
-    @GET
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("user")
-    @RolesAllowed("user")
-    public String getFromUser() {
-        String thisuser = securityContext.getUserPrincipal().getName();
-        return "{\"msg\": \"Hello to User: " + thisuser + "\"}";
```
Comment (Runi-VN) [inline src/main/java/rest/SWAPIRessource.java]: I dont necessarily agree with the deletions here, they are needed for #7

File: src/main/java/facades/ApiFacade.java (hunk 1 above)
Comment (Castau) [reply src/main/java/facades/ApiFacade.java:21]: I was a result of needing to return a "key" along with the result, so google was a good friend :D

File: src/main/java/rest/SWAPIRessource.java (hunk 2 above)
Comment (Castau) [reply src/main/java/rest/SWAPIRessource.java]: Oh, I'm sorry - I thought it was leftover from Maltes last project so i just mindlessly deleted them. I will add them in again

**Review by Runi-VN — APPROVED** [review]
Good job. :)

**Pull request merged**